web/modules/custom/dpl_example_content/dpl_example_content.module

```diff
@@ -63,4 +63,28 @@ function dpl_example_content_modules_installed(array $modules) : void {
   }
 
   $logger->info("Created {$num_instances} opening hours instances");
+
+  // Set page.front link to /frontpage.
+  $config_site = \Drupal::configFactory()->getEditable('system.site');
+  $current_front_page = $config_site->get('page.front');
+
+  $new_front_page = '/frontpage';
+  if ($current_front_page !== $new_front_page) {
+    $config_site->set('page.front', '/frontpage')->save();
+  }
+
+  $logger->info("Update frontpage link to {$new_front_page}, as part of example content installation.");
+
+  // Set example logo.
+  $config_novel = \Drupal::configFactory()->getEditable('novel.settings');
+  $logo_path = 'public://2024-07/logoeksempelbib.png';
```

Comment:
As first I was worried about this path as it seems to be related to the current month and I thought it would be obsolete when we got to august. Now I see that it is probably fixed by the timestamp embedded in the exported media element 😅 .